spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -304,6 +304,11 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
If it's an autoconfigure dependency, you might as well make it a dependency for the project, since autoconfigure is pulled in by all starters. 
I am concerned about a dependency difference between pulling in a module vs pulling in the same module via starter -- in the first case, `commons-lang` will be a test dependency, and in the second it will be a compile dependency. I've also seen IntelliJ do strange things when a dependency was present as both test and compile scope in the hierarchy.

Reply:
I tried to add it only where it's needed. I think it was pulled transitively with something else before.
IntelliJ used to make test dependencies act like compile dependencies, but I'm not sure if it still does it.
In any case, maven build is the source of truth.